curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646e44246c224501ee59542f.md

```diff
@@ -0,0 +1,161 @@
+---
+id: 646e44246c224501ee59542f
+title: Step 45
+challengeType: 0
+dashedName: step-45
+---
+
+# --description--
+
+At the end of `formattedDate` you will split the string using the `split()` method. You will pass the string `'-'` as an argument to the `split()` method.
```

Comment:
Since this is the first time the `split` method is introduced, we should provide a brief explanation. 

```suggestion
In JavaScript, the `split()` method is used to split up a string into an array of substrings. This method also comes with an optional separator parameter which is a type of pattern that tells the computer where each split should happen.

 "coca-cola".split("-") // this would return [ 'coca', 'cola' ]

At the end of `formattedDate`, split the string using the `split()` method and use a `'-'` for the separator. 
```

Reply:
added